uploading_packages/bintray/uploading_bintray.rst

```diff
@@ -53,24 +53,21 @@ Setting the remotes in this way will make your Conan client resolve packages and
 repositories in the following order of priority:
 
   1. `conan-center`_
-  2. `conan-transit`_
-  3. Your own repository
+  2. Your own repository
 
 If you want to have your own repository prioritized, please use the ``--insert`` command line option
 when adding it:
 
 .. code-block:: bash
 
-    $ conan remote add <your_remote> <your_url> --insert
+    $ conan remote add <your_remote> <your_url> --insert 1
```

Comment:
Please check that this work. If indexes are 0 based (and I think they are), this is not inserting in first place, but after conan-center. Default arg value for --insert is 0

Reply:
Yes, it should be ``$ conan remote add <your_remote> <your_url> --insert 1``, thanks!